packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfApplication-CSharp/net6.0/App.xaml.cs

```diff
@@ -0,0 +1,20 @@
+﻿#if (!csharpFeature_ImplicitUsings)
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+#endif
+
+using System.Configuration;
+using System.Data;
+using System.Windows;
+
+namespace Company.WpfApplication1
```

Comment:
Any reason not to support [file-scoped namespaces](https://docs.microsoft.com/dotnet/csharp/whats-new/csharp-10#file-scoped-namespace-declaration)?

Reply:
Working on that currently, marking it draft meanwhile.